website/README.md

```diff
@@ -40,4 +52,7 @@ Once your PR to `stable-website` is merged, open a PR bumping the submodule comm
 
 ### Deployment
 
-New commits in `hashicorp/terraform` and `hashicorp/terraform-cdk` don't automatically deploy the site. To use the latest upstream content, you'll need to open a PR bumping the submodule commit. If your changes aren't being deployed, it's very likely that you need to open a PR to update the submodule commit.
+New commits in `hashicorp/terraform` do not automatically deploy the site. Do the following for documentation pull requests:
+- **Add a backport label to the PR.** Use the label that corresponds to the latest Terraform patch release (e.g., `1.2-backport`). When you merge your PR to `main`, GitHub bot automatically generates a backport PR to merge your commits into the appropriate release branch.
+- **Merge the backport PR.** When all tests pass successfully, merge the backport PR into the release branch. The new content will be added to the site during the next minor release.
```

Comment:
Simply noting / no action needed:

Once we begin displaying multiple versions for `terraform`, merging backport PRs for **older** versions will update their associated versions on the site within the hour.

Backport PRs for the current, latest version will not affect the latest version on the website. This is so that docs changes may be _queued up_ for the next patch release.